src/xform_to_datalog/authorization_logic.cc

```diff
@@ -35,48 +35,47 @@
 
 // Generates datalog facts from the given program.
 extern "C" int generate_datalog_facts_from_authorization_logic(
-  const char* program, const char* program_path,
-  const char* out_dir, const char* decl_skip_vec);
+    const char* input_program, const char* output_file,
+    const char* decl_skip_vec);
 
 #endif
 
 namespace raksha::xform_to_datalog {
 
 int GenerateDatalogFactsFromAuthorizationLogic(
-    absl::string_view program,
-    const std::filesystem::path &program_dir,
-    const std::filesystem::path &result_dir,
-    const std::vector<absl::string_view> &relations_to_not_declare) {
-
+    absl::string_view program, const std::filesystem::path& program_dir,
+    const std::filesystem::path& result_dir,
+    const std::vector<absl::string_view>& relations_to_not_declare) {
```

Comment:
It looks like `clang-format --style=Google` is what changes this. When I try to make it the way you suggest and save it, my text editor runs clang-format and puts it back to the way you don't like. Since, as you mentioned, the Google style guide discourages declaring multiple variables in a single declaration, it seems like the problem you are concerned about is prevented as long as we follow that guidance. What if we just avoid declaring multiple variables in a single line but allow modifiers to be placed next to types so that we can be consistent with both `clang-format --style=Google` and the Google style guide?